environment.yml

```diff
@@ -69,7 +69,7 @@ dependencies:
   - pandas-datareader
   - pyyaml
   - py
-  - pytorch
+  # - pytorch  https://github.com/pandas-dev/pandas/pull/49798
```

Comment:
Now that the pytorch test and ASV are removed, I think it's okay to remove this too